src/condor_vm-gahp/vmgahp_config.cpp

```diff
@@ -40,10 +40,7 @@ operator=(const VMGahpConfig& old)
 	m_vm_type = old.m_vm_type;
 	m_vm_max_memory = old.m_vm_max_memory;
 	m_vm_networking = old.m_vm_networking;
-	m_vm_networking_types.clearAll();
-	if( old.m_vm_networking_types.isEmpty() == false ) { 
-		m_vm_networking_types.create_union(oldp->m_vm_networking_types, false);
-	}
+	m_vm_networking_types = old.m_vm_networking_types;
```

Comment:
Looks like the old code combined any pre-existing m_vm_networking_types with new ones, but the new code unconditionally overwrites the old -- is this want we want?

Reply:
It's a weird thing to be doing in operator=(). And this operator=() is never used in the code, aside from the class's copy constructor (which I believe is also never used).